resources/views/listing/partials/item.blade.php

```diff
@@ -20,19 +20,17 @@ class="mb-3 h-48 w-full rounded-t object-contain"
                 <x-rapidez::no-image v-else class="mb-3 h-48 rounded-t" />
                 <div>
                     <x-rapidez::highlight attribute="name" class="text-base font-medium hover:underline decoration-2 "/>
-                    @if (!Rapidez::config('catalog/frontend/show_swatches_in_product_list'))
-                        <div class="flex items-center space-x-2">
-                            <div class="font-semibold">
-                                @{{ (item.special_price || item.price) | price }}
```

Comment:
See my other comment, just to make sure: the other price is the right one? We don't need this one anymore?